### PR TITLE
SME and QE review and test for Bxmsdoc 3726 master Getting started with business processes

### PR DESCRIPTION
**Pull request by michelehaglund**
Ivo and Kirill, please check the following descriptions in chapter 2. They are two new example projects:

http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3726/#decision-examples-central-con_business-processes-getting-started

**Course_Scheduling:** (Decision management) Example course scheduling and curriculum decision process. Assign lectures to rooms and determines a student’s curriculum based on factors, such as course conflicts and class room capacity.
**Dinner_Party:** (Decision management) Guest seating optimization using guided decision tables. Dinner party seating assignments based on each guest’s job type, political beliefs, and known relationships.

Original Jira: https://issues.jboss.org/browse/BXMSDOC-3726
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3726/

### CODE REVIEW

File: assemblies/assembly_business-processes/main.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ include::_artifacts/author-group.adoc[]
 As a business rules and processes developer, you can use {CENTRAL} in {PRODUCT} to design business processes to meet specific business requirements. {PRODUCT} provides example projects with example business assets directly in {CENTRAL} as a reference. This document describes how to create and test an example mortgage application project based on the *Mortgage_Process* sample project included in {CENTRAL}.
 
 .Prerequisites
-* {EAP_LONG} 7.2.0 is installed. For details, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
+* {EAP_LONG} {EAP_VERSION} is installed. For details, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html-single/installation_guide/[_{EAP_LONG} {EAP_VERSION} Installation Guide_].
```
Comment (csherrar) [inline assemblies/assembly_business-processes/main.adoc]: Should the URL version be /7.2/?

For sentences referring to EAP, which EAP version is preferable? 7.2 {EAP_VERSION} or 7.2.0 {EAP_VERSION_LONG}?

File: doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc [hunk 2]
```diff
@@ -29,3 +29,5 @@ endif::[]
 * *Mortgages*: (Decision management) Example loan approval process using decision assets. Determines loan eligibility based on applicant data and qualifications.
 * *Employee_Rostering*: (Business optimization) Example employee rostering optimization using decision and solver assets. Assigns employees to shifts based on skills.
 * *OptaCloud*: (Business optimization) Example resource allocation optimization using decision and solver assets. Assigns processes to computers with limited resources.
+* *Course_Scheduling*: (Business optimization) Example course scheduling and curriculum decision process. Assign lectures to rooms and determines a student's curriculum based on factors, such as course conflicts and class room capacity.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc]: s/Assign/Assigns

File: doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc [hunk 3]
```diff
@@ -29,3 +29,5 @@ endif::[]
...
+* *Dinner_Party*: (Business optimization) Guest seating optimization using guided decision tables. Dinner party seating assignments based on each guest's job type, political beliefs, and known relationships.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc]: Minor suggestion:
s/Dinner party seating assignments based on each guest's job type, political beliefs, and known relationships./Assigns guest seating based on each guest's job type, political beliefs, and known relationships./

File: doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc (hunk 2 above)
Comment (michelehaglund) [reply doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc]: Made plural. Thanks.

File: doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc (hunk 3 above)
Comment (michelehaglund) [reply doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-con.adoc]: Changed as suggested.

**Pull request merged**